1.process_data/nbconverted/process_jump_pilot.py

```diff
@@ -175,8 +192,14 @@ def fill_dmso(df):
     # Final path of each cell count output file
     output_cell_count_file = f"{output_cell_count_path}/{plate_name}_cellcount.tsv"
 
-    # Path of each normalized out single cell dataset
-    output_file = f"{normalized_path}/{plate_name}_normalized_sc.parquet"
+    # Path template
+    path_template = f"{normalized_path}/{plate_name}"
```

Comment:
Recommend removing this variable if not used. I don't see it being used any time below this.

Reply:
Good catch, forgot to remove this